src/main/java/frc/robot/subsystems/arm/PrototypeArm.java

```diff
@@ -0,0 +1,145 @@
+        elbowMotor.setInverted(ArmConstants.clockWise);
+        elbowMotor.config_kP(0, ArmConstants.elbowP);
+        elbowMotor.config_kI(0, ArmConstants.elbowI);
+        elbowMotor.config_kD(0, ArmConstants.elbowD);
+
+    }
+
+    public void setShoulderJointPower(double power) {
+        shoulderMotor.set(TalonSRXControlMode.PercentOutput, power);
+    }
+
+    public void setElbowJointPower(double power) {
+        elbowMotor.set(TalonSRXControlMode.PercentOutput, power);
+    }
+
+    public void setShoulderJointVelocity(double velocity){
+        shoulderMotor.set(ControlMode.Velocity, velocity);
+    }
+
+    public void setElbowJointVelocity(double velocity){
+        elbowMotor.set(ControlMode.Velocity, velocity);
+    }
+
+    public double getShoulderJointPosition() {
+        return Math.toDegrees(unitModel.toUnits(shoulderMotor.getSelectedSensorPosition()));
+    }
+
+    public void setShoulderJointPosition(double angle) {
+        shoulderMotor.set(ControlMode.MotionMagic, unitModel.toTicks(Math.toRadians(angle)), DemandType.ArbitraryFeedForward, shoulderFeedforward);
+    }
+
+    public double getElbowJointPosition() {
+        return Math.toDegrees(unitModel.toUnits(elbowMotor.getSelectedSensorPosition()));
+    }
+
+    public void setElbowJointPosition(double angle) {
+        elbowMotor.set(ControlMode.MotionMagic, unitModel.toTicks(Math.toRadians(angle)), DemandType.ArbitraryFeedForward, elbowFeedForward);
+    }
+
+
+    public double getShoulderMotorPower() {
+        return shoulderMotor.getMotorOutputPercent();
+    }
+
+    public double getElbowMotorPower() {
+        return elbowMotor.getMotorOutputPercent();
+    }
+
+    public double getShoulderMotorVelocity(){
+        return shoulderMotor.getSelectedSensorVelocity();
+    }
+
+    public double getElbowMotorVelocity(){
+        return elbowMotor.getSelectedSensorVelocity();
+    }
+
+    public void setPosition(Translation2d armLocation){
+        setShoulderJointPosition(kinematics.inverseKinematics(armLocation.getX(), armLocation.getY()).shoulderAngle);
+        setElbowJointPosition(kinematics.inverseKinematics(armLocation.getX(), armLocation.getY()).elbowAngle);
+    }
+
+    public double deadBend(double value){
+        if (Math.abs(value)>0.05) return value;
+        return 0;
+    }
+
+    public String getSubsystemName() {
+        return "PrototypeArm";
+    }
+
+    public void periodic(){
+        timer.reset();
+        timer.start();
+        double shoulderVelocity = getShoulderMotorVelocity();
+        double elbowVelocity = getElbowMotorVelocity();
+        double Time = timer.get();
+        double shoulderAcceleration = (shoulderVelocity - prevShoulderVelocity)/ Math.abs(Time-Time2);
+        double elbowAcceleration = (elbowVelocity - prevElbowVelocity)/Math.abs(Time-Time2);
+        prevShoulderVelocity = shoulderVelocity;
+        prevElbowVelocity = elbowVelocity;
+        Time2 = timer.get();
```

Comment:
But you need the timer object to start it

Reply:
I'm not sure I understood what you meant, but if I did, notice that I used `Timer.getFPGATimestamp()` with a capital T. This is because `getFPGATimestamp()` is a static method, so you don't need to instantiate a `Timer` object to use it.